### ISSUE TITLE
Can't run develop or start.

### ISSUE THREAD

**Issue by Nic128** [donejs/donejs#523]
- Used node 4.2.6/npm 2.14
- Tried donejs v0.6.0 and v0.7.0-pre.1
- donejs init lottery-app
- cd lottery-app
- donejs develop
- I'm on linux

```
can-serve starting on http://localhost:8080
Potentially unhandled rejection [5] Error: Error loading "jquery@2.1.4#jquery" at file:/home/nmartel/repos/dev/lottery-app/node_modules/jquery/jquery.js
Error loading "jquery@2.1.4#jquery" from "@ssr/plugins/canjs" at file:/home/nmartel/repos/dev/lottery-app/node_modules/can-ssr/lib/plugins/canjs.js
ENOENT: no such file or directory, open '/home/nmartel/repos/dev/lottery-app/node_modules/jquery/jquery.js'
    at Error (native)
Potentially unhandled rejection [5] Error: Error loading "jquery@2.1.4#jquery" at file:/home/nmartel/repos/dev/lottery-app/node_modules/jquery/jquery.js
Error loading "jquery@2.1.4#jquery" from "lottery-app@0.0.0#index.stache!done-autorender@0.6.1#autorender" at file:/home/nmartel/repos/dev/lottery-app/src/index.stache
ENOENT: no such file or directory, open '/home/nmartel/repos/dev/lottery-app/node_modules/jquery/jquery.js'
    at Error (native)
```

**Comment by matthewp** [comment]
Thanks, looking into this.

**Comment by matthewp** [comment]
Cause is getting rid of the jquery peerDependency in canjs.  Probably we need to either restore it or make it a regular dependency.

**Comment by matthewp** [comment]
https://github.com/canjs/canjs/pull/2242

**Comment by daffl** [comment]
Fix has been published on NPM as 2.3.15
